src/cstring.rs

```diff
@@ -0,0 +1,41 @@
+use std::{ffi::{self}, fmt};
+
+use serde::{de::{self, Visitor}, Deserializer, Serializer};
+
+
+pub type CString = ffi::CString;
```

Comment:
```suggestion
use std::fmt;
use serde::{de::{self, Visitor}, Deserializer, Serializer};

pub use std::ffi::CString;
```

`pub use` is the proper way to re-export a type.

Reply:
Actually, I don't think this even needs to be re-exported...